test/python/common.py

```diff
@@ -38,6 +38,7 @@ def with_ngraph(self, l, config=tf.ConfigProto()):
                 meta_optimizer_iterations=rewriter_config_pb2.RewriterConfig.
                 ONE,
                 min_graph_nodes=-1,
+                #fail_on_optimizer_errors=True,
```

Comment:
Change 2018 -> 2018-2019

Reply:
Minor : Please add a ToDo to the comment